Add Windows requirements

Dr-Electron opened the pull request:
# Description of change

Refactored the Rust getting started page in regards to the windows requirements

## Type of change

- Documentation Fix

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

Dr-Electron committed:
Add Windows requirements for python bindings

Thoralf-M commented on documentation/docs/getting_started/python.mdx:
```diff
@@ -20,14 +20,28 @@ import SafePasswordStorage from '../_admonitions/_warning_safe_password_storage.
 * [setuptools-rust](https://pypi.org/project/setuptools-rust/) >= 0.10.2
 * Make sure you have fulfilled the [Rust Getting Started](rust.mdx) requirements to compile the binding.
 
+:::info Windows Requirements
```
Would it make more sense to include it to the `rust.mdx` since it's required also for all libs this way?

Dr-Electron replied:
True that 🤔 
Will update the PR later on 👍

Dr-Electron committed:
Refactor Rust getting started

Dr-Electron committed:
Add line highlight

Dr-Electron commented:
I added the prerequisites to the rust page. 
Removed the `Dependency` header as I think it just adds confusion if we link from bindings to `Prerequisites` and people might not look at `Dependency` although that might be needed to.
Removed the openssl and cmake guide as it wasn't needed for me to install those.
Added a one liner for windows to install everything needed.
Added tokio to the example `Cargo.toml` as it is needed in the example afterwards. (Highlighted the `iota-wallet` line).
Fixed the example code

Dr-Electron commented:
Are cmake and openssl still required to be installed? I now tested on macOS too and it doesn't seem to be needed there either. (Openssl was there by default)

Thoralf-M commented:
> Are cmake and openssl still required to be installed? I now tested on macOS too and it doesn't seem to be needed there either. (Openssl was there by default)

I think it's still required

Dr-Electron commented:
Hmm. I definitively didn’t have cmake installed. 🤔

Thoralf-M commented:
> Hmm. I definitively didn’t have cmake installed. thinking

Hm maybe it just needs a C compiler, in the workflow we just install clang for windows https://github.com/iotaledger/wallet.rs/blob/develop/.github/workflows/build-and-test.yml
